### PR TITLE
fix: remove content max-width on dot-step

### PR DESCRIPTION
**Pull request by 07akioni**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [ ] Site / documentation update
- [ ] Demo update
- [x] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Perfermance optimization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Other (about what?)

### 💡 Background and solution
Removing the max-width on dot step.

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English |  Remove the 140px width limit on vertical dot Steps.Step.  |
| 🇨🇳 Chinese |  移除了垂直点状 Steps.Step 的 140px 宽度限制。 |

### ☑️ Self Check before Merge

⚠️ Please check all items below before review. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

**Comment by afc163** [comment]
解决什么问题。

**Comment by zombieJ** [comment]
垂直 dot 样式占了整宽但是展示区域只有 140px

File: components/steps/style/progress-dot.less [hunk 1]
```diff
@@ -47,9 +47,6 @@
         }
       }
     }
-    &-content {
-      width: @steps-desciption-max-width;
```
Comment (zombieJ) [inline components/steps/style/progress-dot.less:51]: 这个 less 变量还有用不？没用的话主题里也删了。

Comment (07akioni) [reply components/steps/style/progress-dot.less:51]: 这个还有用

Comment (07akioni) [reply components/steps/style/progress-dot.less:51]: 对横着的 dot 还有影响

**Review by zombieJ — APPROVED** [review]

**Comment by afc163** [comment]
> 移除了垂直点状 Steps.Step 的 140px 宽度限制。

image

**Pull request merged**